helpers/points.py

```diff
@@ -22,11 +49,8 @@ def is_negative(self) -> bool:
     def is_zero(self) -> bool:
         return self.amount.is_zero()
 
-    def add(self, delta: "Points") -> "Points":
-        return Points(self.amount + delta.amount)
-
-    def subtract(self, delta: "Points") -> "Points":
-        return Points(self.amount - delta.amount)
+    def to_integral(self) -> "Points":
```

Comment:
lmao you could have just do a int(points) i believe

Reply:
You're right, but since Decimal has all sorts of weird rounding rules I wanted to make the rounding explicit.